react/typescript/kitchensink/config/license-header.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2021 Looker Data Sciences, Inc.
```

Comment:
be nice if this could be one copy in the root of your repo

Reply:
It would but I want the ability for a developer to be able to copy the example directory somewhere else and just run with no issues. The downside is some duplication. The build scripts are all duplicated as well. This was the issue we found when Jax initially added hello world to the codegen repo, it was too tied to lerma

I do envisage a script that will keep this things in sync.

I think I can use lerna in this repo but it is more for building and testing thing when a PR is created.

I was playing around with manifests with multiple extensions and they actually work quite nicely, no issues with entitlements (it is only marketplace extensions that have issues). I'm thinking of writing a script that will build looker project from all of the examples and get it into a looker instance. Maybe a hackday project

Reply:
Makes sense. Thanks!